### ISSUE TITLE
Boot Loop after update 21.1.1

### ISSUE THREAD

**Issue by Nope7** [dbeaver/dbeaver#12937]
#### System information: 
- Operating system (distribution) and version: Windows 10 21H1
- DBeaver version: 21.1.1
- Additional extensions: Office plugin

#### Connection specification:
- Database name and version: IBM i DB2
- Driver name: JTOpen
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
After update from 21.1.0 to 21.1.1 (and office plugin installation) dbeaver at the first run, starts to open/close the "starting popup" forever...and it's also impossible to close the program...so I need to disconnect my user to end the bootloop. 

I also tried to uninstall 21.1.1 and install 21.1.0 (that I used until today) but the issues persists.

**Comment by Nope7** [comment]
I also tried with a clean installation (flag on Reset Setting) but it wasn't useful.

**Comment by Nope7** [comment]
I don't know how, but I solved the issue. 

I think that it depended on the fact I was opening the app using a shortcut on the quick launch bar on W10. 

Even if that was effectively pointing to the right path, when I tried to open the program directly from File Explorer (C:\Program\dbeaver\dbeaver.exe) the app started regularly. 

And moreover after a first run from file explorer, also the shortcut on the quick launch could open the program.

So, even if I don't know how could I solve the issue, at the moment I don't have any issue and I'm running the latest version 21.1.1.

Thanks
